### PR TITLE
Carol/doc/SFD Proctoring Followup

### PR DESCRIPTION
**Pull request by catong**
Follow up to initial release of SFD. Minor doc changes to reflect UI changes, screenshot requests, fix alt text.
@srpearce Would you mind doing a quick check? Thanks!

### CODE REVIEW

File: en_us/students/source/SFD_credit_courses/SFD_proctored_exams.rst [hunk 1]
```diff
@@ -237,7 +241,7 @@ operating system, you would see the page shown.
 
 .. image:: /Images/Proctoring_SS_Download.png
  :width: 400
- :alt: The page on which you choose to take an exam with proctoring or without.
+ :alt: The proctoring software dialog with links to download the application for Mac or Windows.
```
Comment (srpearce) [inline en_us/students/source/SFD_credit_courses/SFD_proctored_exams.rst]: dialog box

**Comment by srpearce** [comment]
I think you could make the screen shots even clearer - the two arrows on the software download image confused me. Do you think folks will understand this?
proctoring_ss_download

I'd also make the line just a bit heavier on the other screen shot - one or two points, maybe. Just to make it stand out a little more.

**Comment by srpearce** [comment]
(By the way, I _loooooooove_ drawing attention to the infamous "Using a Windows?" text... Sigh...)

**Comment by srpearce** [comment]
Editing pass complete. Just a couple small comments.

**Comment by catong** [comment]
Squashed commits; merging https://github.com/edx/edx-documentation/pull/512

**Pull request merged**